src/Controllers/ElementSiteTreeFilterSearch.php

```diff
@@ -47,8 +52,13 @@ protected function applyDefaultFilters($query)
                 return false;
             }
 
-            // Check whether the search term exists in the nested page content
-            $pageContent = $siteTree->getElementsForSearch();
+            if ($this->config()->get('render_elements_for_cms_search') === true) {
+                // Check whether the search term exists in the nested page content
+                $pageContent = $siteTree->getElementsForSearch();
+            } else {
+                $pageContent = $siteTree->getContentFromElementsForCmsSearch();
```

Comment:
Given we're just looking at DB fields directly here... is there a way we can craft this as a db query instead of using `stripos`? I suspect that would be faster. Though if that's too complex to do right now we can always merge this PR and look at that as a potential enhancement down the line.

Reply:
Yeah ...... technically.  Though it would involve working out what all of the fields are ahead of time, and then putting all of those into a big `$filter` and putting `:PartialMatch` on all the things. PartialMatch translates to SQL `WHERE <field> LIKE '%<searchterm>%'`